### PR TITLE
Feature/nxos ssh get interfaces counters

### PR DESCRIPTION
**Pull request by ExaneServerTeam**
=Implements nxos_ssh get_interfaces_counters=

* Performs the parsing thought the json pipped output.
* mgmt specific case is taken into account
* test included

### CODE REVIEW

File: napalm/nxos_ssh/nxos_ssh.py [hunk 1]
```diff
@@ -1641,3 +1641,102 @@ def get_optics(self):
...
+                    map_d = if_mapping[if_v]['mapping']
+                    for k, v in map_d.items():
+                        if v in row:
+                            if_counter[k] = int(row[v])
+                    all_stats_d[interface].update(if_counter)
+                    break
+
+        return all_stats_d
+
+        # print(counters_table_raw)
```
Comment (mirceaulinic) [inline napalm/nxos_ssh/nxos_ssh.py:1742]: Please remove

File: napalm/nxos_ssh/nxos_ssh.py [hunk 2]
```diff
@@ -1641,3 +1641,102 @@ def get_optics(self):
...
+        'rx_octets': int,
+        'tx_unicast_packets': int,
+        'rx_unicast_packets': int,
+        'tx_multicast_packets': int,
+        'rx_multicast_packets': int,
+        'tx_broadcast_packets': int,
+        'rx_broadcast_packets': int,
+        """
+        if_mapping = {
+            'eth': {
```
Comment (mirceaulinic) [inline napalm/nxos_ssh/nxos_ssh.py:1663]: I think Black doesn't like this, can you upgrade to the latest version then run `black .`

**Review by mirceaulinic — COMMENTED** [review]
Looks good, just please make sure the tests are passing, then we should be gtg.

File: napalm/nxos_ssh/nxos_ssh.py (hunk 2 above)
Comment (ExaneServerTeam) [reply napalm/nxos_ssh/nxos_ssh.py:1663]: @mirceaulinic  Thanks for the review.
I fix it asap.

Comment (ExaneServerTeam) [reply napalm/nxos_ssh/nxos_ssh.py]: @mirceaulinic Fixes on their way with latest push.

**Review by mirceaulinic — APPROVED** [review]
LGTM. Thanks @ExaneServerTeam!

**Pull request merged**